bin/cheribsdtest/cheribsdtest_ptrace.c

```diff
@@ -133,10 +133,9 @@ CHERIBSDTEST(test_ptrace_readtags, "Basic test of PIOD_READ_CHERI_TAGS")
 {
 	struct ptrace_io_desc piod;
 	pid_t pid;
-	uintcap_t *pp;
+	uintcap_t pp[8] __attribute__((aligned(8*sizeof(uintcap_t)))) = { 0 };
 	char tagbuf[1];
 
-	pp = calloc(8, sizeof(*pp));
```

Comment:
My only thought is that elsewhere I used `aligned_alloc` (in the third test in this file) so have some slight preference for just doing that along with a memset over a stack allocation.  If you want to keep it on the stack you can use the C11 _Alignas() instead of the GCC __attribute__ syntax.  Also, spaces around the `*` operator are preferred.